doc/modules/neural_networks_supervised.rst

```diff
@@ -71,6 +77,14 @@ Classification
 Class :class:`MLPClassifier` implements a multi-layer perceptron (MLP) algorithm
 that trains using `Backpropagation <http://ufldl.stanford.edu/wiki/index.php/Backpropagation_Algorithm>`_.
 
+Following example plot shows first layer weights in a MLPClassifier trained on the
```

Comment:
the statement ?(I kinda derived it from the examples itself), I would appreciate some explanation :)